packages/ui-test-sandbox/src/index.ts

```diff
@@ -22,6 +22,6 @@
  * SOFTWARE.
  */
 
+export type { SinonStub, SinonSpy, SinonSandbox } from 'sinon'
```

Comment:
this is used in some tests also it might be useful for those who use our testing framework